docs/user/build/install/ojdbc.rst

```diff
@@ -1,34 +1,5 @@
 Oracle (Optional)
 -----------------
 
-In order to use the Oracle module you need the proprietary JDBC driver from Oracle.
+As of Feb 287 2020 Oracle has started hosting their JDBC drivers and required dependencies on Maven Central, special actions are no longer required.
```

Comment:
You got long Februaries where you live :-D

Reply:
:rofl: